Content.Server/_Goobstation/MedicalPatch/MedicalPatchSystem.cs

```diff
@@ -0,0 +1,139 @@
+using Content.Shared.Administration.Logs;
+using Content.Server.Medical.Components;
+using Content.Server.Popups;
+using Content.Shared.FixedPoint;
+using Content.Shared.Sticky;
+using Content.Shared.Sticky.Systems;
+using Content.Shared.Sticky.Components;
+using Robust.Shared.Timing;
+using Content.Shared.Chemistry.EntitySystems;
+using Content.Shared.Chemistry;
+using Content.Server.Polymorph.Systems;
+using Content.Shared.Database;
+using Content.Shared.Hands.Components;
+using Content.Shared.Hands.EntitySystems;
+
+namespace Content.Server.Medical;
+
+public sealed class MedicalPatchSystem : EntitySystem
+{
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly StickySystem _stickySystem = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly SharedSolutionContainerSystem _solutionContainers = default!;
+    [Dependency] private readonly ReactiveSystem _reactiveSystem = default!;
+    [Dependency] protected readonly ISharedAdminLogManager _adminLogger = default!;
+    [Dependency] private readonly SharedHandsSystem _hands = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<MedicalPatchComponent, EntityUnstuckEvent>(OnUnstuck);
+        SubscribeLocalEvent<MedicalPatchComponent, EntityStuckEvent>(OnStuck);
+    }
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        if (!_timing.IsFirstTimePredicted)
+            return;
+
+        foreach (var comp in EntityManager.EntityQuery<MedicalPatchComponent>())
```

Comment:
This is absolutely a disaster of an EntityQuery that is made only sensible by there being a relatively small number of this component in the game at any one time. I'll still accept this if only because the sticky visualizer is something that I wanted to code anyway to port sticker books from Aurora, but I want to refactor this when I do.